Sources/Clients/OnLedgerEntitiesClient/OnLedgerEntitiesClient+Live.swift

```diff
@@ -6,101 +6,393 @@ import SharedModels
 
 // MARK: - OnLedgerEntitiesClient + DependencyKey
 extension OnLedgerEntitiesClient: DependencyKey {
+	public static let maximumNFTIDChunkSize = 29
```

Comment:
maybe rename it `limitPerPage`?

Reply:
it is just that in this context it not a page size actually, since the endpoint for the get NFT Data by ids is not a paginated endpoint, rather it accepts a max of 29 ids in the array, the same way state/entity/details accepts at most 20 addresses.